<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTilesetDialog</class>
 <widget class="QDialog" name="NewTilesetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Tileset</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetMinAndMaxSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="tilesetGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Tileset</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>&amp;Name:</string>
        </property>
        <property name="buddy">
         <cstring>name</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="tilesetType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Based on Tileset Image</string>
         </property>
         <property name="toolTip">
          <string>A single image containing a grid of fixed-size tiles</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tileset Atlas</string>
         </property>
         <property name="toolTip">
          <string>A single image with custom tile regions that can be rearranged (useful for varied tile sizes, y-sorting, etc.)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Collection of Images</string>
         </property>
         <property name="toolTip">
          <string>A collection of individual tile images</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="embedded">
        <property name="text">
         <string>Embed in map</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="imageGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Image</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="5">
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>&amp;Browse...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="sourceLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Source:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="useTransparentColor">
        <property name="text">
         <string>Use transparent color:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="Tiled::ColorButton" name="colorButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="generateGrid">
        <property name="text">
         <string>Generate tiles based on grid</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="6">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="Tiled::ExpressionSpinBox" name="tileHeight">
          <property name="suffix">
           <string> px</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>32</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="tileHeightLabel">
          <property name="text">
           <string>Tile height:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="Tiled::ExpressionSpinBox" name="tileWidth">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string extracomment="Remember starting with a space."> px</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>32</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="marginLabel">
          <property name="toolTip">
           <string>The space at the edges of the tileset.</string>
          </property>
          <property name="text">
           <string>Margin:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="Tiled::ExpressionSpinBox" name="margin">
          <property name="suffix">
           <string> px</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="Tiled::ExpressionSpinBox" name="spacing">
          <property name="suffix">
           <string> px</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="tileWidthLabel">
          <property name="text">
           <string>Tile width:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="spacingLabel">
          <property name="toolTip">
           <string>The space between the tiles.</string>
          </property>
          <property name="text">
           <string>Spacing:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLineEdit" name="image"/>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="dropperButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Pick color from image</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/16x16/stock-tool-color-picker-16.png</normaloff>:/images/16x16/stock-tool-color-picker-16.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tiled::ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Tiled::ExpressionSpinBox</class>
   <extends>QSpinBox</extends>
   <header>expressionspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>tilesetType</tabstop>
  <tabstop>embedded</tabstop>
  <tabstop>image</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>useTransparentColor</tabstop>
  <tabstop>colorButton</tabstop>
  <tabstop>dropperButton</tabstop>
  <tabstop>tileWidth</tabstop>
  <tabstop>tileHeight</tabstop>
  <tabstop>margin</tabstop>
  <tabstop>spacing</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>useTransparentColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tryAccept()</slot>
 </slots>
</ui>
